### ISSUE TITLE
question: is it possible to use translation "on demand"? without waiting embedded into next compiled versions

### ISSUE THREAD

**Issue by yilmazdurmaz** [godotengine/godot-editor-l10n#8]
Hi there,

Recently, I have been working on Turkish translations, both proofreading current ones and adding new translations. Too many to go, but I am determined to run for completion if nothing bad happens.

I also have the setup to compile from source on Linux, so I can test my translations when satisfied. (Sometimes It is hard where a string belongs though)

The thing is, I don't know how frequently you are pulling translations from Weblate, and integrating them into the next compilation. One thing for sure is that it is most possible before I conclude my work, which means incomplete translations will go to the next compilations (as they always did previously).

I want to share my work with my friends and whoever else is interested, for a test drive, but the problem is that they most likely will not be using Linux. thus, I cannot simply share with them the one I compiled from the source.

In short, my work does not seem to be available for maybe a month or longer. and the question is: is there a way, like putting a "po" file near the executable, to run with the latest translation I can provide (by downloading files from Weblate directly).

PS: it is a hard-to-find topic if you already have an answer in the documentation, since the "translation" and "localization" words use the search space for "making your game for everyone" context, rather than "how we translate for the engine and editor".
